src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -932,7 +932,9 @@ def prepare_networking(self):
         # in our case when nodes are ready, vips will be there for sure
         if self._ha_not_none():
             vips_info = self.api_client.get_vips_from_cluster(self.id)
-            self.nodes.controller.set_dns(api_ip=vips_info["api_vip"], ingress_ip=vips_info["ingress_vip"])
+            api_ip = vips_info["api_vips"][0].ip() if len(vips_info["api_vips"]) > 0 else ""
```

Comment:
would empty string as IP work, or could this be causing validation issues? Same below

Reply:
that should not change the flow here. if there is a VIP value it that array, it will get passed as `api_ip` and `ingress_ip`.
If there isn't, that was already the case in the existing flow. I just pass an empty string as it would have done anyhow.